packages/react-components/src/components/SendBox.tsx

```diff
@@ -242,9 +248,26 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     // Message can be empty if there is a valid attachment upload
     if (
       sanitizeText(message).length > 0 ||
-      /* @conditional-compile-remove(attachment-upload) */ hasCompletedAttachmentUploads(attachmentsWithProgress)
+      /* @conditional-compile-remove(attachment-upload) */ isAttachmentUploadCompleted(attachmentsWithProgress)
     ) {
-      onSendMessage && onSendMessage(message);
+      onSendMessage &&
+        onSendMessage(
+          message,
+          /* @conditional-compile-remove(attachment-upload) */
+          {
+            attachmentMetadata: attachmentsWithProgress
+              ?.filter((v) => {
```

Comment:
Just curious why `v`?

Reply:
v as in value, i am open for other suggestions as well